### DISCUSSION TITLE
How to raise resources for pods created

### DISCUSSION THREAD

**Discussion by bpopovich44** [rancher/rke2#3974] [General]
Hi have a RKE2 High Availability Cluster running three master nodes and three worker nodes.  The cluster is running great, but on occasions a pod will fail because there is not enough resources on the pod.  How can I manage/change memory and ephemeral storage to each created pod.  Also, can this be limited to a namespace as well or will it be to ALL generated pods.  I did find documentation on this but it is general information.  I don't know which yaml these resources are in on rke2

Thank you

**Comment by brandond** [comment]
Resource requests and limits are configured in the spec of each individual pod.

https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/

**Reply by bpopovich44** [reply]
Ok, thanks Brandon!  You have been alot of help with my cluster.  I will read through this.
